IPython/html/static/base/less/page.less

```diff
@@ -5,16 +5,23 @@
  */
 
 
+html, body {
+    min-height: 100%;
+    height: 100%;
+    overflow: hidden; // Prevents scrollbar in html element
+}
+
 body {
     background-color: @body-bg;
-    /* This makes sure that the body covers the entire window and needs to
-       be in a different element than the display: box in wrapper below */
-    position: absolute;
-    left: 0px;
-    right: 0px;
-    top: 0px;
-    bottom: 0px;
```

Comment:
The part that I understand about this is that it expands the body element to cover the whole window (apparently, an old hack), but this is equivalent to `min-height: 100%; height: 100%;`. However, it was introducing the problem in which the print preview failed to load.